test/minigzip.c

```diff
@@ -276,6 +275,11 @@ int main(int argc, char *argv[]) {
     char *level = "6";
     char *type = "b";
 
+    if ((argc == 1) || (argc == 2 && strstr(argv[1], "-help"))) {
```

Comment:
strstr() is a hack, should use strcmp, otherwise someone could pass in any junk (including real filenames) containing "-help" anywhere... This defeats the purpose even though it would be nice for it to catch any unknown command-line flag, but even that requires checking the first character which strstr() doesn't.

Reply:
Seems like adding help text is more trouble than it is worth..

Reply:
Nah... It's just some functions should never be used... strstr() is one of them...

Reply:
Ok it is changed.